<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TQITTE4" failures="1" tests="1" name="Test2" time="26.235" errors="0" timestamp="2022-08-09T18:14:52 IST">
  <testcase classname="com.crm.sdet.organization.CreateOrganizationTest" name="createOrganization" time="17.194">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.crm.sdet.organization.CreateOrganizationTest.createOrganization(CreateOrganizationTest.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- createOrganization -->
</testsuite> <!-- Test2 -->
